flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/api/status/JobManagerDeploymentConditionStatus.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.flink.kubernetes.operator.api.status;
+
+/** Condition Status of the Flink JobManager Kubernetes deployment. */
+public enum JobManagerDeploymentConditionStatus {
+    READY("True", "JobManagerReady", "JobManager is running and ready to receive REST API calls"),
+    MISSING("False", "JobManagerDeploymentMissing", "JobManager deployment not found"),
+    DEPLOYING("False", "JobManagerIsDeploying", "JobManager process is starting up"),
```

Comment:
is deploying a starting up the same thing?

Reply:
OK that makes sense. Maybe link to that class in a comment to say where you got the descriptions from.